Code/Tools/ProjectManager/Platform/Windows/Application_windows.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Application.h>
+
+#include <DbgHelp.h>
+#include <errorrep.h>
+
+#include <AzCore/IO/FileIO.h>
+
+namespace O3DE::ProjectManager
+{
+    LONG WINAPI CreateMiniDump(struct _EXCEPTION_POINTERS* exceptionPointers)
+    {
+        static const AZStd::string dumpDirectoryAlias{ "@log@/" };
+        static const AZStd::string dumpFileName{ "o3de.dmp" };
+        static const AZStd::string dumpAliasPath{ dumpDirectoryAlias + dumpFileName };
+
+        AZ::IO::FixedMaxPath dumpPath{ dumpAliasPath };
+        if (auto fileIoBase = AZ::IO::FileIOBase::GetInstance(); fileIoBase != nullptr)
+        {
+            dumpPath = fileIoBase->ResolvePath(dumpPath, dumpAliasPath.c_str());
+        }
```

Comment:
Where does the .dmp file go when you comment out the if block on line 25? 

```suggestion
        static const AZStd::string dumpDirectoryAlias{ "@log@/" };
        static const AZStd::string dumpFileName{ "o3de.dmp" };
        static const AZStd::string dumpAliasPath{ dumpDirectoryAlias + dumpFileName };

        AZ::IO::FixedMaxPath dumpPath{ dumpFileName }; // don't use the alias path in case fileIoBase is nullptr
        if (auto fileIoBase = AZ::IO::FileIOBase::GetInstance(); fileIoBase != nullptr)
        {
            dumpPath = fileIoBase->ResolvePath(dumpPath, dumpAliasPath.c_str());
        }
```